src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Get started with telemetry in context
+metaDescription: Get relevant metrics, events, logs, traces, and other telemetry data within the context of your app or host, without additional configuration or context switching in the New Relic One UI.
+---
+
+When you're troubleshooting an issue in your app or host, you need all the New Relic One tools at your fingertips. But you don't want to do a lot of context switching across the UI or be overwhelmed by the wealth of information available.
+
+Our telemetry in context functionality gives you an easy way to troubleshoot problems across your apps, services, other entities, and data types, without having to leave the current UI page you're viewing. We curate the data automatically, including entity GUIDs for related APM service entities, so that highly correlated logs are already visible.
+
+![Telemetry in context](./images/demotron6-logs030722.png "Telemetry in context")
+
+<figcaption>
+  Drill down into your logs, errors, traces, and spans, all from the APM **Summary** page in New Relic One.
+</figcaption>
+
+More importantly, when a problem arises, you immediately have an overall view of the health of **all** your systems, regardless of the type of data in those individual systems and components. But you don't need to waste critical time trying to decide where to start troubleshooting.
+
+We provide a subset of the most relevant data coming from multiple types of telemetry data, so you can quickly focus. This also makes it easier to link your log telemetry data across accounts.
+
+## Get started [#get-started]
+
+Applications don't exist in isolation. The resources you use to monitor, troubleshoot, and improve their performance are both time-critical and complex. You need a simple yet holistic solution to quickly see and act on all your telemetry in context.
+
+Ready to get started?
+
+1. If you don't have one already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
+2. Update to the [supported agent version](#agents) for your apps and hosts.
+3. [Enable](#agents) telemetry in context for your agent, and make a few small updates to your configuration file.
+
+That's it! Now, to start troubleshooting with relevant telemetry data in your logs, go to New Relic One:
+
+* Explorer UI at [one.newrelic.com](https://one.newrelic.com)
+* Explorer UI for EU region data center if applicable: [one.eu.newrelic.com](https://one.eu.newrelic.com)
+
+
+Your logs will automatically include attributes such as `span.id`, `trace.id`, `hostname`, `entity.guid`, and more. These make it easier to query and facet on related telemetry data.
+
+You can also add more attributes to make your troubleshooting focus even more precise. For example, you may want to add more attributes to get other key telemetry data specific to your ecosystem.
+
+## APM agent configuration [#agents]
```

Comment:
How about "Configure telemetry in your agents" or similar? The title feels a little vague.